components/membership/group-policy-test-common/build.gradle

```
@@ -0,0 +1,16 @@
+plugins {
+    id 'corda.common-publishing'
+    id 'corda.common-library'
+}
+
+description 'Group policy provider component'
```

```suggestion
description 'Group policy provider fake component for testing'
```
or something similar?

---

Probably the whole module could be moved from `components/membership/` to `testing/`...
The testing modules lives there.